site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1681,9 +1681,10 @@ public function getBuildStatusOfGradeable(string $gradeable_id): void {
                 $logs = $this->getBuildLogs($gradeable_id);
 
                 $needle = 'The submitty configuration validator detected the above error in your config.';
+                $makeErrorNeedle = 'MAKE ERROR';
                 $haystack = $logs->json['data'][0];
 
-                if (strpos($haystack, $needle) !== false) {
+                if (strpos($haystack, $needle) !== false || strpos($haystack, $makeErrorNeedle) !== false) {
```

Comment:
I would say if it fails to build with error, it shouldn't be warning but alarming. (so that user will know their attempt to build has failed)

Reply:
Changed it, does it look good now?